include/wifi_hal_ap.h

```diff
@@ -23,1069 +23,3130 @@
+    wifi_connection_status_t conn_status; /**< Connection status. */
+    wifi_scan_params_t scan_params; /**< Scan parameters. */
+    wifi_vap_security_t security; /**< Security settings. */
+    mac_address_t mac;        /**< MAC address. */
+    wifi_mld_info_sta_t mld_info; /**< MLD information. */
+} __attribute__((packed)) wifi_back_haul_sta_t;
+
+/**
+ * @brief Maximum length of an SSID.
+ */
+#define WIFI_AP_MAX_SSID_LEN 33
+
+/**
+ * @brief Fronthaul BSS information.
+ */
+typedef struct
+{
+    CHAR ssid[WIFI_AP_MAX_SSID_LEN]; /**< SSID. */
+    BOOL enabled; /**< Whether the BSS is enabled. */
+    BOOL showSsid; /**< Whether to show the SSID. */
+    BOOL isolation; /**< Whether client isolation is enabled. */
+    INT mgmtPowerControl; /**< Management frame power control. */
+    UINT bssMaxSta; /**< Maximum number of stations allowed. */
+    BOOL bssTransitionActivated; /**< Whether BSS transition is activated. */
+    BOOL nbrReportActivated; /**< Whether neighbor report is activated. */
+
+    BOOL rapidReconnectEnable; /**< Whether rapid reconnect is enabled. **This field should not be implemented in the HAL.** */
+    UINT rapidReconnThreshold; /**< Rapid reconnect threshold. **This field should not be implemented in the HAL.** */
+    BOOL vapStatsEnable; /**< Whether VAP statistics are enabled. **This field should not be implemented in the HAL.** */
+
+    wifi_vap_security_t security; /**< Security settings. */
+    wifi_interworking_t interworking; /**< Interworking settings. */
+    wifi_preassoc_control_t preassoc; /**< Pre-association control settings. */
+    wifi_postassoc_control_t postassoc; /**< Post-association control settings. */
+    BOOL mac_filter_enable; /**< Whether MAC filtering is enabled. */
+    wifi_mac_filter_mode_t mac_filter_mode; /**< MAC filter mode. */
+
+    BOOL sec_changed; /**< Whether security settings have changed. **This field should not be implemented in the HAL.** */
+
+    wifi_wps_t wps; /**< WPS settings. */
+    BOOL wmm_enabled; /**< Whether WMM is enabled. */
+    BOOL UAPSDEnabled; /**< Whether U-APSD is enabled. */
+    wifi_bitrate_t beaconRate; /**< Beacon rate. */
+    mac_address_t bssid; /**< The BSSID. This variable should only be used in the get API. It can't be used to change the interface MAC. */
+    UINT wmmNoAck; /**< WMM no-acknowledgement settings. */
+    UINT wepKeyLength; /**< WEP key length. */
+    BOOL bssHotspot; /**< Whether the BSS is a hotspot. */
+    UINT wpsPushButton; /**< WPS push button configuration. */
+    char beaconRateCtl[32]; /**< Beacon rate control string. */
+    BOOL network_initiated_greylist; /**< Whether network-initiated greylisting is enabled. */
+    BOOL mcast2ucast; /**< Whether multicast-to-unicast conversion is enabled. */
+    BOOL connected_building_enabled; /**< Whether connected building is enabled. */
+    wifi_mld_info_ap_t mld_info; /**< MLD information. */
+} __attribute__((packed)) wifi_front_haul_bss_t;
```

Comment:
Two parameters related to wifi_front_hal_bss_t is missing which should be added after mld_info 

    BOOL   hostap_mgt_frame_ctrl;
    BOOL   mbo_enabled;

Reply:
ok will check